WooCommerce/WooCommerceTests/POS/Controllers/POSEntryPointControllerTests.swift

```diff
@@ -6,7 +6,7 @@ struct POSEntryPointControllerTests {
     @Test func eligibilityState_is_set_to_eligible_when_i2_feature_is_disabled() async throws {
```

Comment:
This test is a bit confusing... why do we set it to eligible if the feature's disabled?